src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -31,18 +161,30 @@ Interaction::Interaction(const std::shared_ptr<ngraph::Node>& op, const dnnl::en
         IE_THROW(NotImplemented) << errorMessage;
     }
     errorPrefix = "Interaction node with name '" + getName() + "'";
+    const auto interaction = std::dynamic_pointer_cast<const InteractionNode>(op);
+    const std::vector<float>& scales = interaction->get_output_scales();
+    if (!scales.empty()) {
+        fqScales = scales;
+        outputDataType  = InferenceEngine::details::convertPrecision(interaction->get_fq_output_type());
+    }
 }
 
-void Interaction::initSupportedPrimitiveDescriptors() {
-    if (!supportedPrimitiveDescriptors.empty())
-        return;
+void Interaction::getSupportedDescriptors() {
```

Comment:
Why you moved precision initialization logic inside `getSupportedDescriptors`. It should be kept inside `initSupportedPrimitiveDescriptors`.

Reply:
getSupportedDescriptors is needed only for oneDNN based nodes where they init `descs` vector: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/fullyconnected.cpp#L550. For your particular node it is not required so it is better to leave the method empty.